module/Motion/BonusBlockCarrier.cpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file BonusBlockCarrier.cpp
+ * @brief ボーナスブロックをベースエリアまで運搬する
+ * @author sugaken0528
+ */
+
+#include "BonusBlockCarrier.h"
+
+void BonusBlockCarrier::carryBonusBlock()
```

Comment:
クラス名で何をするのか分かるし，`run`の方が楽でよくない？
関数を複数個に分ける想定ならこのままでもいいけど，分けるとしてもLRだけかな？
その場合は`runLeft`とか`runLeftCourse`とか？
```suggestion
void BonusBlockCarrier::run()
```